bundles/org.openhab.binding.huesync/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="huesync"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!--
+		Documentation:
+		- https://www.openhab.org/docs/developer/addons/config-xml.html
+		- https://www.openhab.org/docs/developer/bindings/thing-xml.html
+		- https://www.openhab.org/schemas/config-description-1.0.0.xsd
+	-->
+
+	<channel-type id="device-info-firmware">
+		<item-type>String</item-type>
+		<label>Firmware Version</label>
+		<description>Installed firmware version</description>
+		<category>text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="device-info-firmware-available">
+		<item-type>String</item-type>
+		<label>Latest Firmware Version</label>
+		<description>Latest available firmware version</description>
+		<category>text</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="connection-name">
+		<item-type>String</item-type>
+		<label>Friendly name</label>
+		<description>Friendly name of the HDMI connection</description>
+		<category>text</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="connection-status">
+		<item-type>String</item-type>
+		<label>HDMI connection Status</label>
+		<description>Status of the HDMI input</description>
+		<category>status</category>
+		<state readOnly="true"></state>
+		<command>
+			<options>
+				<option value="unplugged"></option>
+				<option value="plugged"></option>
+				<option value="linked"></option>
+				<option value="unknown"></option>
+			</options>
+		</command>
+	</channel-type>
+
+	<channel-type id="connection-type">
+		<item-type>String</item-type>
+		<label>Friendly type</label>
+		<description>Type of the connected HDMI device</description>
+		<category>text</category>
+		<state readOnly="true"></state>
+		<command>
+			<options>
+				<option value="generic"></option>
+				<option value="video"></option>
+				<option value="game"></option>
+				<option value="music"></option>
+				<option value="xbox"></option>
+				<option value="playstation"></option>
+				<option value="nintendoswitch"></option>
+				<option value="phone"></option>
+				<option value="desktop"></option>
+				<option value="laptop"></option>
+				<option value="appletv"></option>
+				<option value="roku"></option>
+				<option value="shield"></option>
+				<option value="chromecast"></option>
+				<option value="firetv"></option>
+				<option value="diskplayer"></option>
+				<option value="settopbox"></option>
+				<option value="satellite"></option>
+				<option value="avreceiver"></option>
+				<option value="soundbar"></option>
+				<option value="hdmiswitch"></option>
+			</options>
+		</command>
+	</channel-type>
+
+	<channel-type id="connection-lastSyncMode">
```

Comment:
Both channel id's and type id's should be lower-case-hyphen
For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention